govcd/system.go

```diff
@@ -18,6 +18,9 @@ import (
 // settings parameter. The settings variable is defined in types.go.
 // Method will fail unless user has an admin token.
 // API Documentation: https://code.vmware.com/apis/220/vcloud#/doc/doc/operations/POST-CreateOrganization.html
+// Organization creation has two bugs https://bugzilla.eng.vmware.com/show_bug.cgi?id=2177355, https://bugzilla.eng.vmware.com/show_bug.cgi?id=2228936 which requires,
+// that creating organization elements are in incorrect order, at least one element(DelayAfterPowerOnSeconds, DeployedVMQuota, StoredVmQuota, UseServerBootSequence, getVdcQuota)
+// is needed then providing generalOrgSettings and VAppLeaseSettings/VAppLeaseSettings provided fully.
```

Comment:
Please rephrase this comment to be clearer and answer the following questions:
- What is the gist of these bugs?
- Which "one element" is needed? It's hard to understand as it stands right now.
- How these two bugs have been worked-around by your code below to make it work?
- The links are useful only if you're connected to internal network. Please remove the links, but keep the issue numbers (BZ 2177355 and BZ 2228936).

Reply:
Comment updated with Giuseppe's help.

Reply:
And Organization creation works. I verified with terraform provider too.